gui_agents/agents/Backend/LybicSDKBackend.py

```
@@ -0,0 +1,355 @@
+
+log = logging.getLogger(__name__)
+
+
+class LybicSDKBackend(Backend):
+    """
+    基于官方Lybic Python SDK的Backend实现
+    支持与LybicBackend相同的Action类型，但使用官方SDK替代HTTP调用
+    """
+    
+    _supported = {Click, DoubleClick, Move, Drag, TypeText, Scroll, Hotkey,
+                  Wait, Screenshot, Memorize}
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)